ethereum/spec/src/main/java/tech/pegasys/teku/spec/executionengine/ExecutionEngineChannel.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.spec.executionengine;
+
+import java.util.Optional;
+import org.apache.tuweni.bytes.Bytes32;
+import org.web3j.protocol.core.methods.response.EthBlock;
+import org.web3j.protocol.core.methods.response.EthBlock.Block;
+import tech.pegasys.teku.infrastructure.async.SafeFuture;
+import tech.pegasys.teku.infrastructure.events.ChannelInterface;
+
+public interface ExecutionEngineChannel extends ChannelInterface {
+  ExecutionEngineChannel NOOP =
+      new ExecutionEngineChannel() {
+        @Override
+        public SafeFuture<Optional<Block>> getPowBlock(Bytes32 blockHash) {
+          return SafeFuture.completedFuture(Optional.empty());
+        }
+
+        @Override
+        public SafeFuture<Block> getPowChainHead() {
+          throw new UnsupportedOperationException();
+        }
+      };
+
+  SafeFuture<Optional<Block>> getPowBlock(Bytes32 blockHash);
+
+  SafeFuture<EthBlock.Block> getPowChainHead();
```

Comment:
I'm not really a fan of exposing the web3j types here.  Would it be possible to have these return `PowBlock`?